community/wmutils/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+cd core || exit 1
```

Comment:
We don't need to do `|| exit` at all as we have `-e` in the shebang (set -e). The shell will do just this on its own.